### PR TITLE
AF-1113: Display Git external URLs on Project Settings

### PR DESCRIPTION
**Pull request by tiagobento**
See: https://issues.jboss.org/browse/AF-1113

1113-gif

### CODE REVIEW

**Comment by tiagobento** [comment]
@ederign Could you please take a took?

**Comment by tiagobento** [comment]
jenkins execute full downstream build

**Comment by tiagobento** [comment]
jenkins execute full downstream build

**Comment by tiagobento** [comment]
jenkins execute full downstream build

**Comment by tiagobento** [comment]
jenkins execute full downstream build

**Comment by tiagobento** [comment]
jenkins execute full downstream build

**Comment by ederign** [comment]
@tiagobento could you please fix the conflicts?

**Comment by tiagobento** [comment]
@ederign Resolved.

File: kie-wb-common-screens/kie-wb-common-project-editor/kie-wb-common-project-editor-api/src/main/java/org/kie/workbench/common/screens/projecteditor/model/ProjectScreenModel.java [hunk 1]
```diff
@@ -27,9 +29,32 @@
 @Portable
 public class ProjectScreenModel {
 
+    @Portable
+    public static class GitUrl {
```
Comment (ederign) [inline kie-wb-common-screens/kie-wb-common-project-editor/kie-wb-common-project-editor-api/src/main/java/org/kie/workbench/common/screens/projecteditor/model/ProjectScreenModel.java]: could you please extract this to an external class?

**Pull request merged**